packages/dashboard-core-plugins/src/redux/actions.ts

```diff
@@ -0,0 +1,115 @@
+import deepEqual from 'deep-equal';
+import { updateDashboardData } from '@deephaven/dashboard';
+import { ThunkAction } from 'redux-thunk';
+import { RootState } from '@deephaven/redux';
+import { Action } from 'redux';
+import { getLinksForDashboard } from './selectors';
+import { FilterSet } from '../panels';
+import { Link } from '../linker/LinkerUtils';
+
+/**
+ * Set the session wrapper for the dashboard specified
+ * @param id The ID of the dashboard to set the session for
+ * @param sessionWrapper The session wrapper object to set for the dashboard
+ */
+export const setDashboardSessionWrapper = (
+  id: string,
+  sessionWrapper: unknown
```

Comment:
We should be able to set the type for `sessionWrapper` here. 
Also need to delete the old `actions.js`, and convert `selectors.js` to TS as well.